pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -72,8 +72,8 @@ const (
 	PodToleratesNodeNoExecuteTaintsPred = "PodToleratesNodeNoExecuteTaints"
 	// CheckNodeLabelPresencePred defines the name of predicate CheckNodeLabelPresence.
 	CheckNodeLabelPresencePred = "CheckNodeLabelPresence"
-	// checkServiceAffinityPred defines the name of predicate checkServiceAffinity.
-	checkServiceAffinityPred = "checkServiceAffinity"
+	// CheckServiceAffinityPred defines the name of predicate checkServiceAffinity.
+	CheckServiceAffinityPred = "CheckServiceAffinity"
```

Comment:
will that introduce backward compatibility?

Reply:
Should be fine. According to: https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/factory/plugins.go#L207-L218
The name of this predicate is not part of scheduler policy configure file.

Reply:
But there could be existing policy config files that are already using the old name.

Reply:
@bsalamat @k82cn AFAIK, the right way of using `serviceAffinity` in policy file is like below:
```
    "apiVersion": "v1",
    "kind": "Policy",
    "predicates": [
        {
            "name": "NoVolumeZoneConflict"
        },
        {
            "name": "MaxEBSVolumeCount"
        },
        {
            "name": "MaxGCEPDVolumeCount"
        },
        {
            "name": "MatchInterPodAffinity"
        },
        {
            "name": "NoDiskConflict"
        },
        {
            "name": "GeneralPredicates"
        },
        {
            "name": "PodToleratesNodeTaints"
        },
        {
            "name": "CheckNodeMemoryPressure"
        },
        {
            "name": "CheckNodeDiskPressure"
        },
        {
            "argument": {
                "serviceAffinity": {
                    "labels": [
                        "region"
                    ]
                }
            },
            "name": "Region"
        }
    ],
```

That's why I claim 
> The name of this predicate is not part of scheduler policy configure file

Since `serviceAffinity` is a `argument` which is not touched by this PR.

Correct me if I misunderstood sth :)

Reply:
Ah, I see. Thanks for clarification.